examples/BME280/BME280.ino

```diff
@@ -123,7 +123,23 @@ void setup() {
   digitalWrite(LED_BUILTIN, PIN_STATE_HIGH);
   adapter = new WebThingAdapter("weathersensor", WiFi.localIP());
 
+  // Set unit for temperature to "celsius". Other option "fahrenheit"
   weatherTemp.unit = "celsius";
+  
+  // Set title to "Pressure"
+  weatherPres.title = "Pressure";
+  // Set unit for pressure to hPa
+  weatherPres.unit = " hPa";
+  // Set pressure to read only (otherweise you may change the values in the gateway interface
+  weatherPres.readOnly = "true";
+  
+  // Set title to "Humidity"
+  weatherHum.title = "Humidity";
+  // Set unit for humidity to %
+  weatherHum.unit = "percent";
```

Comment:
Since you made this a LevelProperty, you should also set minimum and maximum (0 and 100).